[2025-07-15T03:45:22.087+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T03:45:22.119+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflights_etl_pipeline.extract_task.extract2 manual__2025-07-15T03:45:20.830159+00:00 [queued]>
[2025-07-15T03:45:22.127+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflights_etl_pipeline.extract_task.extract2 manual__2025-07-15T03:45:20.830159+00:00 [queued]>
[2025-07-15T03:45:22.154+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-15T03:45:22.186+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_task.extract2> on 2025-07-15 03:45:20.830159+00:00
[2025-07-15T03:45:22.197+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1811) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-15T03:45:22.199+0000] {standard_task_runner.py:63} INFO - Started process 1826 to run task
[2025-07-15T03:45:22.198+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'airflights_etl_pipeline', 'extract_task.extract2', 'manual__2025-07-15T03:45:20.830159+00:00', '--job-id', '1727', '--raw', '--subdir', 'DAGS_FOLDER/airflight_pipeline.py', '--cfg-path', '/tmp/tmpcs87_h_5']
[2025-07-15T03:45:22.200+0000] {standard_task_runner.py:91} INFO - Job 1727: Subtask extract_task.extract2
[2025-07-15T03:45:22.218+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-15T03:45:22.259+0000] {task_command.py:426} INFO - Running <TaskInstance: airflights_etl_pipeline.extract_task.extract2 manual__2025-07-15T03:45:20.830159+00:00 [running]> on host 5a1293588d06
[2025-07-15T03:45:22.608+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='airflights_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_task.extract2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T03:45:20.830159+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T03:45:20.830159+00:00'
[2025-07-15T03:45:22.609+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T03:45:22.621+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T03:45:22.622+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: extract_etl() missing 1 required positional argument: 'output_file_full'
[2025-07-15T03:45:22.631+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=airflights_etl_pipeline, task_id=extract_task.extract2, run_id=manual__2025-07-15T03:45:20.830159+00:00, execution_date=20250715T034520, start_date=20250715T034522, end_date=20250715T034522
[2025-07-15T03:45:22.665+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1727 for task extract_task.extract2 (extract_etl() missing 1 required positional argument: 'output_file_full'; 1826)
[2025-07-15T03:45:22.694+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-15T03:45:22.709+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T03:45:22.713+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
